Classes/Repository/RepositoryIssuesViewController.swift

```diff
@@ -94,6 +99,19 @@ IndicatorInfoProvider {
             }
         }
     }
+    func composeIndicatorTitle(with itemCount: Int) -> String? {
+        var newTitle: String
+        switch type {
+        case .issues: newTitle = "Issues (\(itemCount))"
```

Comment:
Shouldn't "Issues" and "Pull Requests" be localised?

Reply:
I did localize the output from `composeIndicatorTitle` method before assigning it to the title  
  ```self?.title = NSLocalizedString(updatedTitle, comment: "")```

Reply:
I think it would be safer and give a better overview if we'd do so here, instead of where it is used.